common/djangoapps/course_modes/tests/test_admin.py

```diff
@@ -178,7 +179,7 @@ def test_verification_deadline_after_upgrade_deadline(self):
 
     def _configure(self, mode, upgrade_deadline=None, verification_deadline=None):
         """Configure course modes and deadlines. """
-        course_mode = CourseMode.objects.create(
+        course_mode = CourseModeFactory(
```

Comment:
The current convention in the codebase is to use Factory.create, I think because it helps IDEs pick up on the types easier.